tests/cases/distributed/catalog/schema.sql

```diff
@@ -0,0 +1,33 @@
+CREATE SCHEMA test_public_schema;
+
+CREATE SCHEMA test_public_schema;
+
+SHOW DATABASES LIKE '%public%';
+
+USE test_public_schema;
+
+CREATE TABLE hello(i BIGINT TIME INDEX);
+
+DROP TABLE hello;
+
+CREATE TABLE hello(i BIGINT TIME INDEX);
```

Comment:
check `SHOW TABLES FROM test_public_schema` and `SHOW TABLES FROM public` here to prevent table being accidentally created on default schema.